H_w 3/task_1(2).rb

```diff
@@ -0,0 +1,24 @@
+i = 1
+
+while i< 11
+
+  x = rand(0..9)
+
+  if x.even?
+    print"#{i}. The number: #{x}, is even, "
```

Comment:
print is a method
the string is an argument
when we call a method with an argument(s) we should either place a space between method and first arg or wrap all args inside a brackets:

method_name arg1, arg2

or 

method_name(arg1, arg2)